### PR TITLE
Increase precision of inferred regularity rates

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@weiglszonja How about this?

### CODE REVIEW

**Comment by weiglszonja** [comment]
Yes, that will also be needed, also https://github.com/catalystneuro/roiextractors/pull/212 is more consistent to `spikeinterface`. I'm rerunning the tests now, but so far it seems good.

**Review by weiglszonja — APPROVED** [review]

File: tests/test_behavior/test_movie_interface.py [hunk 1]
```diff
@@ -248,11 +249,10 @@ def test_movie_regular_timestamps(self):
             Movie=dict(starting_times=self.starting_times, timestamps=timestamps, external_mode=True)
         )
 
-        rate = 1.0 / np.round(timestamps[1] - timestamps[0], 9)
         fps = 25.0
         expected_warn_msg = (
-            f"The fps={fps} from movie data is unequal to the difference in "
-            f"regular timestamps. Using fps={rate} from timestamps instead."
+            "The fps=25.0 from movie data is unequal to the difference in "
+            "regular timestamps. Using fps=5.0 from timestamps instead."
```
Comment (weiglszonja) [inline tests/test_behavior/test_movie_interface.py:255]: @CodyCBakerPhD 
Looking at the CI run, this should either be fps=5.000000000000007, or should use rate = 1.0 / (timestamps[1] - timestamps[0]), but I can see why you removed that before, but hopefully how rate is calculated won't be changed soon.

Comment (CodyCBakerPhD) [reply tests/test_behavior/test_movie_interface.py:255]: Right, that's just a display issue though(with the print statement). Fixing now

**Review by weiglszonja — APPROVED** [review]
Thank you @CodyCBakerPhD for helping with this.

**Pull request merged**